hypothesis-python/tests/ghostwriter/test_expected_output.py

```diff
@@ -0,0 +1,133 @@
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis/
+#
+# Most of this work is copyright (C) 2013-2020 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+"""
+'Golden master' tests for the ghostwriter.
+
+To update the recorded outputs, simply run the tests with the
+"GHOSTWRITER_UPDATE" environment variable set.
```

Comment:
Could we do this via a pytest flag instead? We can add those in our conftest.

Reply:
Done, with a new `--hypothesis-update-outputs` argument.